tests/e2e_logging.py

```diff
@@ -1588,6 +1588,61 @@ def test_committed_index(network, args):
     }
 
 
+@reqs.description("Test the commit endpoints view_history feature")
+def test_commit_view_history(network, args):
+    remote_node, _ = network.find_primary()
+    with remote_node.client() as c:
+        # the original endpoint should still work
+        res = c.get("/app/commit")
+        assert res.status_code == http.HTTPStatus.OK
+        assert "view_history" not in res.body.json()
+        txid = res.body.json()["transaction_id"]
+
+        # non-true view_history should still work
+        res = c.get("/app/commit?view_history=nottrue")
```

Comment:
This seems surprising, it would perhaps be more user friendly to return a 400 that says "Invalid value, view_history must be one of true, false".

Reply:
Changed to be `true` or `false` and return 400 for others.